packages/dcos-log/extra/dcos-log-master.service

```diff
@@ -1,5 +1,5 @@
 [Unit]
-Description=DC/OS Log service master: DC/OS logging service
+Description=DC/OS Log Master: exposes component, container, and task logs
```

Comment:
You can hit AdminRouter on the master to get redirected to the agent's log service:
https://github.com/dcos/adminrouter/blob/master/nginx.master.conf#L317
The log service itself just exposes the journald logs from that node, which on a master does not include any tasks.